### PR TITLE
Populating Components pages metadata

### PR DESCRIPTION
**Pull request by halocline**
#### What does this PR do?

- To expand searchable surface, populates the sections and tags metadata for Components pages.
- Also opportunistic clean up of examples throwing console warnings

Related to https://github.com/grommet/hpe-design-system/issues/1849

#### Where should the reviewer start?

data/structures/components.js

#### What are the relevant issues?

Related to https://github.com/grommet/hpe-design-system/issues/1849

### CODE REVIEW

File: aries-site/src/data/structures/components.js [hunk 1]
```diff
@@ -28,12 +28,21 @@ export const components = [
...
+      'Anchor in a form',
+      'Disabled Anchor',
+      'Anchor with weight',
+    ],
     preview: {
       component: () => <AnchorPreview />,
       background: 'background-front',
     },
     relatedContent: ['Button', 'Typography', 'Forms'],
+    tags: ['hyperlinks', 'links', 'linking', 'navigation'],
```
Comment (britt6612) [inline aries-site/src/data/structures/components.js:45]: would adding `href` be an option ?

File: aries-site/src/data/structures/components.js [hunk 2]
```diff
@@ -205,12 +312,13 @@ export const components = [
...
     seoDescription:
       'Checkbox component for HPE Design System. UX usage examples and guidance on how to ensure a checkbox maintains accessiblity.',
-    sections: [],
+    sections: ['Toggle', 'CheckBox with Description'],
     relatedContent: ['CheckBoxGroup', 'Forms', 'RadioButtonGroup', 'Select'],
     preview: {
       component: () => <CheckBoxPreview />,
       background: 'background-front',
     },
+    tags: ['check box', 'toggle', 'input type'],
```
Comment (britt6612) [inline aries-site/src/data/structures/components.js:321]: `Indeterminate`

**Review by britt6612 — APPROVED** [review]
Left 2 comments otherwise Looks Good! 
Thanks for cleaning up other files

File: aries-site/src/data/structures/components.js (hunk 2 above)
Comment (halocline) [reply aries-site/src/data/structures/components.js:321]:
```suggestion
    tags: ['check box', 'toggle', 'input type', 'indeterminate'],
```

**Pull request merged**